calculator/src/main/java/com/market/banica/calculator/handler/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.market.banica.calculator.handler;
+
+import com.market.banica.common.exception.IncorrectResponseException;
+import com.market.banica.common.exception.TrackingException;
+import io.grpc.StatusException;
+import io.grpc.StatusRuntimeException;
+import org.modelmapper.spi.ErrorMessage;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+
+@ControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
+
+
+    @ExceptionHandler({IncorrectResponseException.class, TrackingException.class})
+    public ResponseEntity<ErrorMessage> customExceptionHandler(Exception exception) {
+        return new ResponseEntity<>(new ErrorMessage(exception.getMessage()), HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+
+    @ExceptionHandler(IllegalArgumentException.class)
+    public ResponseEntity<ErrorMessage> illegalArgumentExceptionExceptionHandler(IllegalArgumentException exception) {
+        return new ResponseEntity<>(new ErrorMessage(exception.getMessage()), HttpStatus.BAD_REQUEST);
+    }
+
+    @ExceptionHandler(Exception.class)
```

Comment:
The response here looks as same as the one in customExceptionHandler method. Isn't it possible to add Exception class in its ExceptionHandler annotation

Reply:
I fix it and add the Exception class in the custom exception handler @ExceptionHandler annotation.